website/pages/docs/secrets/venafi/index.mdx

```diff
@@ -0,0 +1,270 @@
+---
```

Comment:
You also need to reference this page in the index for it to show up in the sidebar. e.g. near https://github.com/hashicorp/vault/blob/c4c15e2cf254007e2d5cb3c40e0ce8f08d409a60/website/data/docs_detailed_categories.yml#L92

Reply:
Also, you can double check that it worked by viewing the preview link for the PR (it does take a little while to build):

![image](https://user-images.githubusercontent.com/211880/77789051-95ef3380-701f-11ea-9e26-68a6f1214169.png)

Reply:
@kalafut it looks like that index was added in vault 1.4, should I rebase to that?

Reply:
Merging out from master to your branch is fine. We'll squash at the end anyway.

Reply:
Thank you @kalafut, I think I found the right place to update.  It was a different file (website/data/docs-navigation.js) but the nav looks correct in the preview.

Reply:
oh, ha... I realize now I had an old branch checked out when I grabbed the link. Glad you found it!